openshift/templates/agent/agent-deploy.bc.prod.param

```diff
@@ -12,14 +12,14 @@
 # SUFFIX=-bc
 # APP_NAME=orgbook
 # APP_GROUP=aries-vcr
-APPLICATION_DOMAIN=orgbook-agent.apps.silver.devops.gov.bc.ca
-APPLICATION_ADMIN_DOMAIN=orgbook-agent-admin.apps.silver.devops.gov.bc.ca
+APPLICATION_DOMAIN=agent.orgbook.gov.bc.ca
+APPLICATION_ADMIN_DOMAIN=agent-admin.orgbook.gov.bc.ca
 # ROUTE_TIMEOUT=120s
 # IMAGE_NAMESPACE=8ad0ea-tools
 # NAMESPACE_NAME=8ad0ea
-TAG_NAME=bc-prod
+TAG_NAME=prod
 # AGENT_LABEL_NAME=OrgBook BC Agent
-AGENT_BASE_URL=https://orgbook-agent.apps.silver.devops.gov.bc.ca
+AGENT_BASE_URL=https://agent.orgbook.gov.bc.ca
```

Comment:
This shoul dbe `https://bc-reg-agent.orgbook.gov.bc.ca`

Reply:
updated